draft-ietf-ppm-dap.md

```diff
@@ -293,8 +293,8 @@ The basic unit of DAP is the "task" which represents a single measurement
 includes the following parameters:
 
 * The type of each measurement.
-* The aggregation function to compute (e.g., sum, mean, etc.) and an optional
-  aggregation parameter.
+* The aggregation function to compute (e.g., sum, mean, etc.) and optional
+  task-constant parameters for the aggregation function.
```

Comment:
This parameter changes in Poplar, so I don't think this definition is correct.

Reply:
That is actually the point I wanted to make here. Previously it looked like the aggregation parameter (the thing that changes in Poplar) was part of the task definition. Since it changes, that's not possible. But of course other things that do not change can be given here.

Reply:
Yeah, the current language seems wrong too. Maybe we could say something like this?

```
* The aggregation function to compute (e.g., sum, mean, etc.) and a way to
  produce optional parameters for the aggregation function.
```

Reply:
Maybe we should just decomplicate things and remove it entirely and assume that this kind of thing is part of "the function".

Reply:
I think removing it entirely is the right way to go.